src/app-layer-parser.c

```diff
@@ -935,6 +935,11 @@ void AppLayerParserTransactionsCleanup(Flow *f, const uint8_t pkt_dir)
     SCLogDebug("start min %"PRIu64, min);
     bool skipped = false;
     // const bool support_files = AppLayerParserSupportsFiles(f->proto, f->alproto);
+    size_t tofree_nb = 0;
```

Comment:
I think this doesn't match the commit message, and should go into its own commit with an extensive explanation